src/stories/input/DatePicker.stories.tsx

```diff
@@ -1,8 +1,8 @@
 import React from "react";
 import { Story, Meta } from "@storybook/react";
+import DateFnsUtils from "@date-io/date-fns";
 import DatePicker, { DatePickerProps } from "../../components/Pickers/DatePicker/DatePicker";
 import { PickersUtilsProvider } from "../../components/PickersUtilsProvider/PickersUtilsProvider";
```

Comment:
Shouldn't this one be changed as well?

(`../../components/PickersUtilsProvider`, and default import)